cmake/modules/RootMacros.cmake

```diff
@@ -34,17 +34,15 @@ else()
   set(runtimedir ${CMAKE_INSTALL_PYTHONDIR})
 endif()
 
+set(ROOT_LIBRARY_PROPERTIES_NOVER ${ROOT_LIBRARY_PROPERTIES_NOVER}
+    SUFFIX ${libsuffix}
+    PREFIX ${libprefix} )
 if(soversion)
-  set(ROOT_LIBRARY_PROPERTIES ${ROOT_LIBRARY_PROPERTIES}
+  set(ROOT_LIBRARY_PROPERTIES ${ROOT_LIBRARY_PROPERTIES} ${ROOT_LIBRARY_PROPERTIES_NOVER}
       VERSION ${ROOT_VERSION}
-      SOVERSION ${ROOT_MAJOR_VERSION}.${ROOT_MINOR_VERSION}
-      SUFFIX ${libsuffix}
-      PREFIX ${libprefix} )
+      SOVERSION ${ROOT_MAJOR_VERSION}.${ROOT_MINOR_VERSION} )
 else()
-  set(ROOT_LIBRARY_PROPERTIES ${ROOT_LIBRARY_PROPERTIES}
-      SUFFIX ${libsuffix}
-      PREFIX ${libprefix}
-      IMPORT_PREFIX ${libprefix} )
+  set(ROOT_LIBRARY_PROPERTIES ${ROOT_LIBRARY_PROPERTIES} ${ROOT_LIBRARY_PROPERTIES_NOVER})
```

Comment:
Hi @stephanlachnit, thanks a lot for the PR!

I have only one remark: what is the reason for removing the `IMPORT_PREFIX ${libprefix}` in the `else(soversion)` branch?

@bellenot, do you happen to know why this was only present in the `else()` but not in the `if(soversion)`?

Personally, I wouldn't remove the `IMPORT_PREFIX` in the same commit or even PR, because this change is unrelated to the goal of having a `ROOT_LIBRARY_PROPERTIES_NOVER` for PyROOT.